sdk/core/azure-core/CMakeLists.txt

```diff
@@ -116,10 +116,9 @@ create_code_coverage(core azure-core azure-core-test)
 
 target_link_libraries(azure-core INTERFACE Threads::Threads)
 
-if(MSVC)
```

Comment:
How do we know when to use `MSVC` and when to use `WIN32`? What's the difference here? I thought there are more configurations where `MSVC` is true but `WIN32` isn't.

Reply:
Because the way our code is written, on Windows platform it would use crypt32. Potentially, not only on MSVC, on any copliler on Windows platform you would be capable to use that dll.